packages/core_unix/core_unix.v0.14.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+version: "v0.14.0"
```

Comment:
```suggestion
```

Reply:
Thanks. I wasn't sure why or whether this field was needed, but I cargo-culted it from somewhere.